packages/oca/tsconfig.json

```diff
@@ -27,7 +25,7 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "ESNext" /* Specify what module code is generated. */,
+    "module": "CommonJS" /* Specify what module code is generated. */,
```

Comment:
I think you can keep this as "ESNext" unless it has caused issues in reuse. I think I tested it in Bifold and it didn't seem to cause an issue.

Reply:
I used the same that AFJ core package used. ESNext gave me some runtime errors.